src/atldld/utils.py

```diff
@@ -111,6 +115,7 @@ def get_image(image_id, folder=None, expression=False):
 
     # Create full path
     additional_specifier = "_expression" if expression else ""
+    additional_specifier += f"_{downsample}"
```

Comment:
Maybe the line below (where the path is constructed as a bare string) is the reason for the bug under point 2 in #41? I feel like we should use `pathlib.Path` here (or `os.path`, but personally I prefer `pathlib`)

So, personally, I would write something like this:
```python
file_name = f"{image_id}-{downsample}"
if expression:
    file_name += "-expression"
path = pathlib.Path(folder) / (file_name + ".jpg")
```

Reply:
Yeh, I might as well do the fix in this PR. It is tiny:) Thanks.

Reply:
Sorry, I changed my mind:D Let's implement this in a separate PR.

Reply:
ha ha, fair enough. Is there already an issue for this? Maybe we can copy the code snippet above over there.

Reply:
Yes, feel free to put it in #41. I think using `pathlib` is a necessity.